src/interceptors/android/adb-commands.ts

```diff
@@ -310,10 +310,12 @@ export async function injectSystemCertificate(
 
                 # Apps inherit the Zygote's mounts at startup, so we inject here to ensure all newly
                 # started apps will see these certs straight away:
-                for Z_PID in "$ZYGOTE_PID $ZYGOTE64_PID"; do
+                for Z_PID in "$ZYGOTE_PID" "$ZYGOTE64_PID"; do
                     # We use 'echo' below to trim spaces
-                    nsenter --mount=/proc/$(echo $Z_PID)/ns/mnt -- \
-                        /bin/mount --bind /system/etc/security/cacerts /apex/com.android.conscrypt/cacerts
+                    if [ -n "$Z_PID" ]; then
+                        nsenter --mount=/proc/$(echo $Z_PID)/ns/mnt -- \
```

Comment:
In fact, since we're now checking `-n` here and using quotes properly, we can probably drop this `echo` and the corresponding comment above, right?